### PR TITLE
Feat/6

### PR DESCRIPTION
**Pull request by bell-person-ii**
1페이지 기능 구현 완료 및 테스트 성공 확인

#########################################################################################

1페이지 응답 형식

=======>>

 "isSuccess": true,
    "code": "COMMON200",
    "message": "성공입니다",
    "result": {
        "mainBanners": {
            "main-banner0": {
                "brand-id": 6,
                "banner-profile": "https://umc.my.test.s3.ap-northeast-2.amazonaws.com/7a7b75ce-ed01-49d2-8496-436773bd3924.png"
            },
            "main-banner1": {
                "brand-id": 7,
                "banner-profile": "https://umc.my.test.s3.ap-northeast-2.amazonaws.com/bf23e730-d957-4cde-be49-9f09b0388e34.png"
            },
            "main-banner2": {
                "brand-id": 5,
                "banner-profile": "https://umc.my.test.s3.ap-northeast-2.amazonaws.com/16a79654-48c0-469c-8e6f-33d7f1bedfe1.png"
            },
            "main-banner3": {
                "brand-id": 4,
                "banner-profile": "https://umc.my.test.s3.ap-northeast-2.amazonaws.com/713ce0e7-0832-4407-a499-0b11e8a9ea25.png"
            },
            "main-banner4": {
                "brand-id": 3,
                "banner-profile": "https://umc.my.test.s3.ap-northeast-2.amazonaws.com/dd579fb9-d154-4a42-a519-bb26d91ed229.png"
            }
        },
        "subBanners": {
            "sub-banner0": {
                "contents-id": 12,
                "banner-img": "https://umc.my.test.s3.ap-northeast-2.amazonaws.com/865d3743-0a81-454b-af83-afd49ff7cba4.png"
            },
            "sub-banner1": {
                "contents-id": 11,
                "banner-img": "event file"
            },
            "sub-banner2": {
                "contents-id": 10,
                "banner-img": "event file"
            },
            "sub-banner3": {
                "contents-id": 9,
                "banner-img": "event file"
            },
            "sub-banner4": {
                "contents-id": 8,
                "banner-img": "event file"
            },
            "sub-banner5": {
                "contents-id": 7,
                "banner-img": "event file"
            },
            "sub-banner6": {
                "contents-id": 6,
                "banner-img": "event file"
            },
            "sub-banner7": {
                "contents-id": 5,
                "banner-img": "event file"
            },
            "sub-banner8": {
                "contents-id": 4,
                "banner-img": "event file"
            },
            "sub-banner9": {
                "contents-id": 3,
                "banner-img": "event file"
            }
        },
        "brandList": {
            "brand0": {
                "brand-id": 1,
                "brand-name": "brand-name 1",
                "brand-image": "https://umc.my.test.s3.ap-northeast-2.amazonaws.com/12b100e0-6221-4af5-8899-975ef5f46755.png",
                "brand-fan": 1
            }
        }
    }
}

### CODE REVIEW

File: src/main/java/com/brandol/controller/BrandController.java
Comment (seungheon123) [inline src/main/java/com/brandol/controller/BrandController.java:1]: Controller에서 swagger 문서 작성까지 해주세요

File: src/main/java/com/brandol/service/BrandService.java
Comment (seungheon123) [inline src/main/java/com/brandol/service/BrandService.java:1]: Transactional(readOnly = true)를 BrandService 최상단에 하고 서비스 로직 중에서 readonly가 아닌것에 Transactional 붙이는 걸로 수정해주세요

**Review by seungheon123 — COMMENTED** [review]
나머지는 다 잘한거 같아

File: src/main/java/com/brandol/service/BrandService.java
Comment (bell-person-ii) [reply src/main/java/com/brandol/service/BrandService.java:1]: 적용 완료요~

File: src/main/java/com/brandol/controller/BrandController.java
Comment (bell-person-ii) [reply src/main/java/com/brandol/controller/BrandController.java:1]: 적용 완료

File: src/main/java/com/brandol/service/BrandService.java
Comment (seungheon123) [reply src/main/java/com/brandol/service/BrandService.java:1]: 다른 Service단에 있는 Transactional도 수정 가능한가요?

Comment (bell-person-ii) [reply src/main/java/com/brandol/service/BrandService.java:1]: 다 클래스 기본은 값은 readOnly로 할까요? 나머지는 조회만 몰려있진 않아서 일단 readOnly 를 메서드 단위로 걸어두긴함

Comment (bell-person-ii) [reply src/main/java/com/brandol/service/BrandService.java:1]: 트랙젝션 디폴트 값 ReadOnly 설정 및 Exception Hanlder 구현 완료하였습니다!!

**Review by seungheon123 — COMMENTED** [review]
굿굿

**Review by seungheon123 — APPROVED** [review]

**Pull request merged**